core/modules/installer/installer.pages.inc

```diff
@@ -921,6 +921,7 @@ function installer_browser_filters_form_submit($form, &$form_state) {
   }
   else {
     $_SESSION['installer_browser_text_filter_' . $type] = $form_state['values']['search_text'];
+    unset($_GET['page']);
```

Comment:
Could you add an `if (isset($_GET['page']))` check here? Otherwise this would throw a notice if "page" is not specified in the URL when the "Clear" button is clicked.

Also odd, shouldn't this be in the `if ($form_state['clicked_button']['#parents'][0] == 'clear')` check? The page would be ignored if clicking the "Clear" button right?